src/themes/index.story.css

```diff
@@ -0,0 +1,29 @@
+z-body {
+  font-size: 16px;
+  letter-spacing: 0.3px;
+}
+
+ul {
+  display: flex;
+  flex-direction: column;
+  width: 400px;
+  padding: 0;
+}
+
+ul > li {
+  display: flex;
+  margin: 4px;
+  justify-content: space-between;
+}
+
+ul > li > code {
+  margin-right: 8px;
+}
+
+ul > li > span {
+  width: 100px;
+  height: 20px;
+  display: inline-block;
+  border-radius: 4px;
+  box-shadow: 0px 0px 3px 0px;
```

Comment:
the shadow is a little bit messy, IMO it's better to use a `border` like
```css
border: 1px solid var(--gray400);
```

Reply:
ok